cms/static/cms/js/modules/cms.modal.js

```diff
@@ -578,7 +556,10 @@ var CMS = window.CMS || {};
                     // check if the item is the last one
                     var last = (index >= breadcrumbs.length - 1) ? 'active' : '';
                     // render breadcrumbs
-                    crumb += '<a href="' + item.url + '" class="' + last + '"><span>' + item.title + '</span></a>';
+                    crumb += template
+                        .replace('{1}', item.url)
+                        .replace('{2}', last)
```

Comment:
potentially after this replace there could be multiple `{3}` in the string
kinda unlikely, but yeah